Bottom: 3bfb0757feb61d6d6462acb0255a3f943413176c
Top:    15dcb194f76023712c70da26fa1e8a1e294dbf2a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-28 00:14:46 +0100

crosshair.c: Remove comment which is no longer relevant


---

diff --git a/src/crosshair.c b/src/crosshair.c
index 296f999..a7942b6 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -712,15 +712,6 @@ CrosshairOff (void)
     }
 }
 
-/*
- * The parameter to HideCrosshair() and RestoreCrosshair() dictates whether 
- * the object you're dragging should be drawn or not.
- *
- * This argument is _not_ saved in the stack, so whether you have drawings
- * following the cursor around or not is dependant on the parameter passed 
- * LAST to either of these two functions.
- */
-
 /* ---------------------------------------------------------------------------
  * saves crosshair state (on/off) and hides him
  */
